v2/jdbc-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/JdbcToSpannerOptions.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.options;
+
+import com.google.cloud.teleport.metadata.TemplateParameter;
+import org.apache.beam.sdk.options.Default;
+
+/** Interface used by the JdbcToSpanner pipeline to accept user input. */
+public interface JdbcToSpannerOptions extends CommonTemplateOptions {
+
+  @TemplateParameter.Text(
+      order = 1,
+      optional = false,
+      regexes = {"^.+$"},
+      groupName = "Source",
+      description = "Comma-separated Cloud Storage path(s) of the JDBC driver(s)",
+      helpText = "The comma-separated list of driver JAR files.",
+      example = "gs://your-bucket/driver_jar1.jar,gs://your-bucket/driver_jar2.jar")
+  String getDriverJars();
+
+  void setDriverJars(String driverJar);
+
+  @TemplateParameter.Text(
+      order = 2,
+      optional = false,
+      regexes = {"^.+$"},
+      groupName = "Source",
+      description = "JDBC driver class name",
+      helpText = "The JDBC driver class name.",
+      example = "com.mysql.jdbc.Driver")
+  String getDriverClassName();
+
+  void setDriverClassName(String driverClassName);
+
+  @TemplateParameter.Text(
+      order = 3,
+      optional = false,
+      regexes = {
+        "(^jdbc:[a-zA-Z0-9/:@.?_+!*=&-;]+$)|(^([A-Za-z0-9+/]{4}){1,}([A-Za-z0-9+/]{0,3})={0,3})"
+      },
+      groupName = "Source",
+      description = "JDBC connection URL string.",
+      helpText =
+          "The JDBC connection URL string. For example, `jdbc:mysql://some-host:3306/sampledb`. Can"
+              + " be passed in as a string that's Base64-encoded and then encrypted with a Cloud"
+              + " KMS key. Note the difference between an Oracle non-RAC database connection string"
```

Comment:
If we are not supporting Oracle, better to remove the wording. Also will help to tell which source we are supporting.

Reply:
Fixed